README.md

```diff
@@ -11,6 +11,24 @@ As part of this repo, we provide the following examples:
 ## Installation
 
 Each tool require a different installation process. TODO: document the general process HERE.
+#### Python Script
+##### Requirements:
+
+- Python version: 2.7
+
+##### Installation instructions:
+  1. Download the repo
+  2. Create the file credentials.py into config/
```

Comment:
It would be easier if the instruction was something like: rename the file config/credentials.py.example to config/credentials.py